extruct/_extruct.py

```diff
@@ -1,169 +1,185 @@
 import logging
 import warnings
 
+from extruct.dublincore import DublinCoreExtractor
 from extruct.jsonld import JsonLdExtractor
-from extruct.rdfa import RDFaExtractor
-from extruct.w3cmicrodata import MicrodataExtractor
-from extruct.opengraph import OpenGraphExtractor
 from extruct.microformat import MicroformatExtractor
-from extruct.dublincore import DublinCoreExtractor
-from extruct.uniform import _umicrodata_microformat, _uopengraph, _udublincore
+from extruct.opengraph import OpenGraphExtractor
+from extruct.rdfa import RDFaExtractor
+from extruct.uniform import _udublincore, _umicrodata_microformat, _uopengraph
 from extruct.utils import parse_xmldom_html
+from extruct.w3cmicrodata import MicrodataExtractor
 
 logger = logging.getLogger(__name__)
-SYNTAXES = ['microdata', 'opengraph', 'json-ld', 'microformat', 'rdfa', 'dublincore']
+SYNTAXES = ["microdata", "opengraph", "json-ld", "microformat", "rdfa", "dublincore"]
 
 
-def extract(htmlstring,
-            base_url=None,
-            encoding="UTF-8",
-            syntaxes=SYNTAXES,
-            errors='strict',
-            uniform=False,
-            return_html_node=False,
-            schema_context='http://schema.org',
-            with_og_array=False,
-            **kwargs):
+def extract(
+    htmlstring,
+    base_url=None,
+    encoding="UTF-8",
+    syntaxes=SYNTAXES,
+    errors="strict",
+    uniform=False,
+    return_html_node=False,
+    schema_context="http://schema.org",
+    with_og_array=False,
+    **kwargs
+):
     """htmlstring: string with valid html document;
```

Comment:
This looks a bit strange as it's not aligned with the rest of the params below. We can move this to the new line.